test/testsuites/opsrcinvalidendpointtest.go

```diff
@@ -0,0 +1,82 @@
+package testsuites
```

Comment:
Should each of these be in their own test suite? All tests focus on invalid `operator sources` which I would argue should be included together in a `InvalidOperatorSourceTestSuite`.

Reply:
Sounds good to me. I will put these tests into a single test-suite

Reply:
These could even go in the `opsrc-creation-test-suite`.

Reply:
I considered that, but I didn't need the operatorsource that is created as part of the TestGroup. Although, it doesn't seem as though the `opsrccreationtests.go` uses it either

Reply:
>I considered that, but I didn't need the operatorsource that is created as part of the TestGroup. 
That's a good point 

>Although, it doesn't seem as though the opsrccreationtests.go uses it either

The test suite runs a series of tests that depend on the `OpSrc` being created before executing.

Reply:
Ah, that's right. I think it still makes more sense to have the invalid tests be a separate testsuite since they don't use the same operatorsource and because they test a different use case.